proglearn/voters.py

```diff
@@ -137,10 +154,11 @@ def _finite_sample_correction(posteriors, num_points_in_partition, num_classes):
         return posteriors
 
 
-class KNNClassificationVoter(BaseVoter):
+class KNNClassificationVoter(BaseVoter, BaseEstimator, ClassifierMixin):
```

Comment:
You don't need to inherit from BaseEstimator. By nature of inheriting from BaseVoter, it inherits from BaseEstimator.